src/include/parser/create_statement.h

```diff
@@ -174,6 +181,8 @@ struct ColumnDefinition {
   std::vector<char*>* foreign_key_source = nullptr;
   std::vector<char*>* foreign_key_sink = nullptr;
 
+  std::vector<char *>* multi_unique_cols = nullptr;
```

Comment:
Just wondering, why not use smart pointer like std::unique_ptr in the functions, so that deletion will be automatically handled.